config/initializers/pagy.rb

```diff
@@ -0,0 +1,212 @@
+# frozen_string_literal: true
+
+# Pagy initializer file (9.0.5)
+# Customize only what you really need and notice that the core Pagy works also without any of the following lines.
+# Should you just cherry pick part of this file, please maintain the require-order of the extras
+
+# Pagy Variables
+# See https://ddnexus.github.io/pagy/docs/api/pagy#variables
+# You can set any pagy variable as a Pagy::DEFAULT. They can also be overridden per instance by just passing them to
+# Pagy.new|Pagy::Countless.new|Pagy::Calendar::*.new or any of the #pagy* controller methods
+# Here are the few that make more sense as DEFAULTs:
+Pagy::DEFAULT[:limit] = 5                    # default
```

Comment:
I think setting a global default of 5 records might not be a good idea 🤔 It's very little and it's better to keep the default more sensible and override them only when needed. Let's set it to what was before (20?) and explicitly set `limit: 5` when we call `pagy` in our controller.

Reply:
If you think this is better, it will be like that:

![obraz](https://github.com/user-attachments/assets/6e2cdb4c-694e-4d41-8906-fbb2a340243d)